### DISCUSSION TITLE
Support for Vue2.x?

### DISCUSSION THREAD

**Discussion by Thatisgreat** [bcakmakoglu/vue-flow#21] [Q&A] [answered]
I'm considering using this awesome component for the existing Vue2.x project, but I found that this component is written in Vue3 and is incompatible. 
Do you think it will be compatible with Vue2.x? 
thank you

**Comment by bcakmakoglu** [comment, marked as answer]
@a290079770 
I'm sorry to disappoint you but Vueflow is not and will not be compatible with Vue2.x
It was built with Vue3 in mind, using features that are Vue3 only, which would make re-writing the library to support Vue2 a real pain and I don't have the time and patience to do it, sorry. 😅
